### PR TITLE
MRC-2050: Implement calibrate submit endpoint

### PR DESCRIPTION
**Pull request by r-ash**
This builds on #233  and should be merged into mrc-2040 after that PR

### CODE REVIEW

File: tests/testthat/test-endpoints-model.R [hunk 1]
```diff
@@ -357,168 +357,3 @@ test_that("Debug endpoint errors on nonexistant id", {
                scalar("Task '1234' not found"))
   expect_equal(error$status_code, 400)
 })
-
-test_that("can calibrate a model result", {
```
Comment (r-ash) [inline tests/testthat/test-endpoints-model.R:361]: These have moved to `test-calibrate` and been updated

File: R/endpoints.R [hunk 2]
```diff
@@ -158,6 +158,23 @@ model_result <- function(queue) {
...
+    verify_result_available(queue, id)
+    calibration_options <- jsonlite::fromJSON(input)
+    if (!is_current_version(calibration_options$version)) {
+      hintr_error(t_("CALIBRATE_SUBMIT_OLD"), "VERSION_OUT_OF_DATE")
+    }
+    tryCatch(
+      list(id = scalar(queue$submit_calibrate(queue$result(id),
+                                              calibration_options$options))),
+      error = function(e) {
+        hintr_error(e$message, "FAILED_TO_QUEUE")
```
Comment (richfitz) [inline R/endpoints.R:172]: when does this happen?

**Review by richfitz — COMMENTED** [review]
This looks super tidy - just one comment about failure to queue. I can't immediately see where that would be thrown but as it's not showing up as a coverage gap it must be?

File: R/endpoints.R (hunk 2 above)
Comment (richfitz) [reply R/endpoints.R:172]: Ah, re: coverage - we don't have coverage enabled here. We have a workflow for tokens now

**Review by richfitz — APPROVED** [review]

**Pull request merged**